android/libraries/rib-base/src/main/java/com/badoo/ribs/clienthelper/Connectable.kt

```diff
@@ -0,0 +1,9 @@
+package com.badoo.ribs.clienthelper
+
+import com.jakewharton.rxrelay2.Relay
+
+interface Connectable<Input, Output> {
```

Comment:
Should we add some simple realisations for avoiding Nothing? on client code, if there is no Input or Output

Reply:
Is there anything wrong with Nothing though? :) It's expressive. Btw we can generate empty sealed classes for Input / Output in templates, that should be enough that you don't need to care, no?